pkg/task/verify.go

```diff
@@ -41,12 +41,12 @@ type TaskResultVerify struct {
 	Split string `json:"split"`
 }
 
-// taskResultVerifyError represent task result verification errors only
-type taskResultVerifyError struct {
+// TaskResultVerifyError represent task result verification related error
```

Comment:
Mark this entire file along with pkg/task/query.go as deprecated.
The `post` property of a RunTask removes the need for verify.go & query.go file present in pkg/task namespace